internal/v1/server_test.go

```diff
@@ -424,6 +424,50 @@ func TestGetComposeMetadata404(t *testing.T) {
 	require.Contains(t, body, "Compose not found")
 }
 
+func TestGetComposes(t *testing.T) {
+	composes := ComposesResponse{
+		Data: []ComposesResponseItem{
+			{
+				CreatedAt: "",
+				Id:        UUIDTest,
+			},
+		},
+	}
+
+	api_srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+
+		err := json.NewEncoder(w).Encode(composes)
+		require.NoError(t, err)
+
+	}))
+	defer api_srv.Close()
+
+	// insert a compose in the mock database
+	dbase := tutils.InitDB()
+	err := dbase.InsertCompose(UUIDTest, "500000", "000000", json.RawMessage("{}"))
+	require.NoError(t, err)
+	created_at, _ := dbase.GetCompose(UUIDTest, "500000")
```

Comment:
- camelcase variables
- check the error here as well
- GetCompose returns a ComposeEntry, so the variable name is a bit confusing.

Reply:
Thanks, I've fixed almost everything. I'll open another PR to test  query parameters and add negative scenarios